physics/GFS_surface_generic_pre.F90

```diff
@@ -87,8 +87,8 @@ subroutine GFS_surface_generic_pre_run (nthreads, im, levs, vfrac, islmsk, isot,
         real(kind=kind_phys), dimension(:), intent(inout) :: sigmaf, work3, zlvl
 
         ! Stochastic physics / surface perturbations
-        real(kind=kind_phys), dimension(:),   intent(out) :: drain_cpl
```

Comment:
It doesn't appear that these fields aren't used. Can we remove them?

Reply:
Thanks for pointing this out. Changed the PR to draft while working on getting these changes through and tested. 
Related to [ccpp-physics #495](https://github.com/NCAR/ccpp-physics/issues/495)